### DISCUSSION TITLE
Welcome to kedro-dvc Discussions!

### DISCUSSION THREAD

**Discussion by shaunc** [FactFiber/kedro-dvc#5] [Announcements]
## 👋 Welcome!
We’re using Discussions to moot plans, and as a place to connect with other members of our community. 

As we start off, we have used discussions to post our plans:

* [Stage 1][disc-stage-1] Kedro pipelines -> DVC pipelines and DVC experiment tracking -> Kedro
* [Stage 2][disc-stage-2] Code, as well as data, should determine what aspects of pipelines are rebuilt
* [Stage 3][disc-stage-3] Distribute Kedro-DVC pipelines

[disc-stage-1]: https://github.com/FactFiber/kedro-dvc/discussions/6
[disc-stage-2]: https://github.com/FactFiber/kedro-dvc/discussions/7
[disc-stage-3]: https://github.com/FactFiber/kedro-dvc/discussions/8

Beyond that, we hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by antonymilne** [comment]
Hello! 👋  I'm Antony and I'm a maintainer on Kedro. I've been trying to learn a bit more about DVC the last couple of weeks to understand better how Kedro and DVC could work together. Very excited for this plugin! Thank you for all the hard work you've done already on it. I love the carefully structured design and planning issues you've set up - it's all amazingly organised!

**Comment by shaunc** [comment]
Thank you Antony! Let's see if we can follow through .... -- and thanks, everywhere, for your comments!
